drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -129,7 +157,7 @@ function DragNDropTemplates(url_name) {
             var item_description = h(
                 'div',
                 { id: item_description_id, className: 'sr' },
-                gettext('Correctly placed in: ') + getZoneTitle(item.zone, ctx)
+                gettext('Correctly placed in: ') + (zone.title || "Unknown Zone")
```

Comment:
@itsjeyd I'll just remove it, since it's more complex than needed here, where we already have the `zone` object available. Thanks for pointing that out.